pkg/helm/client/client.go

```diff
@@ -98,16 +102,19 @@ func NewRESTClientGetter(mgr manager.Manager, ns string) (genericclioptions.REST
 
 var _ kube.Interface = &ownerRefInjectingClient{}
 
-func NewOwnerRefInjectingClient(base kube.Client, ownerRef metav1.OwnerReference) kube.Interface {
+func NewOwnerRefInjectingClient(base kube.Client, restMapper meta.RESTMapper,
```

Comment:
Is it need to be exported?

Reply:
This PR did not change the export functionality.

Reply:
It is currently exported, so this PR is not changing anything related to that. In retrospect, this probably should have never been exported, but it is, so I think we should leave it that way.

Reply:
Huge chunks of the Helm operator code will soon be migrated to a new implementation and a more usable set of libraries, so I don't want us to be too worried about small things like this with the existing code.

Reply:
@camilamacedo86 Please let me know if this clarifies your concerns